lib/commands/find.js

```diff
@@ -58,9 +60,16 @@ helpers.findElOrEls = async function (strategy, selector, mult, context = '') {
     }
   }
 
-  return element;
+  if (mult) {
```

Comment:
should we double-check that ios has this logic? and really it seems like common logic we could extend from basedriver somehow. but that can wait for another time.